server/routes/visitJourney/visitJourneyUtils.test.ts

```diff
@@ -8,3 +9,56 @@ describe('getUrlPrefix', () => {
     expect(getUrlPrefix(true, 'ab-cd-ef-gh')).toBe('/visit/ab-cd-ef-gh/update')
   })
 })
+
+describe('validationErrorsMojAlert', () => {
+  const prisonerName = 'Smith, John'
+  const visitStartTimestamp = '2022-03-12T09:30:00'
+
+  it('should return a MojAlert item from 422 error `APPLICATION_INVALID_NON_ASSOCIATION_VISITS`', () => {
+    const validationErrors: ApplicationValidationErrorResponse['validationErrors'] = [
+      'APPLICATION_INVALID_NON_ASSOCIATION_VISITS',
+    ]
+
+    const { mojAlert, url } = validationErrorsMojAlert(prisonerName, visitStartTimestamp, validationErrors)
+
+    expect(mojAlert).toStrictEqual({
+      text: 'Select a new visit time.',
+      showTitleAsHeading: true,
+      title: 'Smith, John now has a non-association on 12 March.',
```

Comment:
Design shows prisoner name as `FIRSTNAME LASTNAME`. It also shows it in this format below the `H1` too.

We have another ticket coming up I believe about prisoner name format. So, worth checking if we should stick with what you have for now and change it all later; or, change it here in this PR. Could be fiddly because we just store `prisoner.name` in the session 🤔

Reply:
we receive prisoner.name at the start of the journey (prisoner profile page) in that layout, think it's a little out of scope to re-build the name differently on 2 separate pages to satisfy the design

https://dsdmoj.atlassian.net/browse/VB-5147 - will be addressed as part of this